README.md

```diff
@@ -12,19 +12,41 @@ pip install python-switchbot
 
 ## Usage
 
-### Devices
+
+### API V1.1 Init
+```python
+from switchbot import SwitchBot
+
+import string
+import random
+
+# To get the token and secret, please refer to https://github.com/OpenWonderLabs/SwitchBotAPI#getting-started
+your_switch_bot_token = '98a6732b2ac256d40ffab7db31a82f518969f4d1a64eadff581d45e902327b7c577aa6ead517bda589c19b4ca0b2599b'
+your_switch_bot_secret = '222cdc22f049d111c5d0071c131b8b77'
+some_random_chars = ''.join(random.choice(string.ascii_lowercase) for i in range(10))
+
+switchbot = SwitchBot(token=your_switch_bot_token, secret=your_switch_bot_secret, nonce=some_random_chars, ver="1.1")
+```
+
+### API V1.0 Init
+
 ```python
 from switchbot import SwitchBot
 
 # To get the token, please refer to https://github.com/OpenWonderLabs/SwitchBotAPI#getting-started
 your_switch_bot_token = '98a6732b2ac256d40ffab7db31a82f518969f4d1a64eadff581d45e902327b7c577aa6ead517bda589c19b4ca0b2599b'
 switchbot = SwitchBot(token=your_switch_bot_token)
+```
+
+### Devices
+```python
 
```

Comment:
```suggestion

### Devices
```python
import uuid

from switchbot import SwitchBot

# To get the token and secret, please refer to https://github.com/OpenWonderLabs/SwitchBotAPI#getting-started
your_switch_bot_token = '98a6732b2ac256d40ffab7db31a82f518969f4d1a64eadff581d45e902327b7c577aa6ead517bda589c19b4ca0b2599b'
your_switch_bot_secret = '222cdc22f049d111c5d0071c131b8b77'
switchbot = SwitchBot(token=your_switch_bot_token, secret=your_switch_bot_secret, nonce=str(uuid.uuid4()))
```

Reply:
According to the official document, they seem to suggest using uuid as nonce.